django_tenants/models.py

```diff
@@ -145,6 +147,16 @@ def get_primary_domain(self):
         except get_tenant_domain_model().DoesNotExist:
             return None
 
+    def reverse(self, request, view_name):
```

Comment:
I would rather leave view_name on first position, to not be changing API of this method, and made request optional keyarg